python_modules/libraries/CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing Libraries
+Welcome! We appreciate your interest in contributing to Dagster Integrations.
+This guide outlines the steps for community members who wish to contribute to
+integratiosn to the project.
+
+## Dagster Libraries
+Dagster Libraries are integrations that can be installed separately from the
+main Dagster package, and provide additional functionality with external tools,
+services, and APIs.
+
+Dagster Libraries may be maintained by either the **Dagster Team** or by
+**Dagster Community** members. Dagster libriaries will include a
+`integration.yaml` in each subfolder, which define, among other things,
+the maintainer of each package.
+
+## integration.yaml
+The `integration.yaml` file must include the following:
+
+```
+package-name: The name of the python package, for example, `dagster-duckdb`
```

Comment:
I think in all current cases we just replace underscores with hyphens to get the package name. If that's the case, maybe we should omit this degree of freedom until we have a reason to add it?

Reply:
I like that